packages/cli/src/commands/generate/dbAuth/dbAuth.js

```diff
@@ -48,7 +48,14 @@ export const files = ({ _tests, typescript }) => {
   const scaffoldOutputPath = path.join(getPaths().web.src, 'scaffold.css')
   if (!fs.existsSync(scaffoldOutputPath)) {
     const scaffoldTemplate = generateTemplate(
-      path.join('scaffold', 'templates', 'assets', 'scaffold.css.template'),
+      path.join(
+        __dirname,
+        '..',
```

Comment:
No need to change, but when you join you can use slashes, and path will do the conversion :)

Reply:
Oooo you can do that with just `.join()`? I thought that was what `.normalize()` was for?

Reply:
I guess join calls normalise underneath! But yeah you can do that